### DISCUSSION TITLE
my kerner says 4

### DISCUSSION THREAD

**Discussion by itsthemoneyman** [3kh0/ext-remover#1180] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
what do i do kryptosmite

**Comment by HelloIloveGimkit2** [comment, marked as answer]
you can't

cryptosmite only works for kernver's 2 and lower

**Comment by HelloIloveGimkit2** [comment]
if you just wanna boot Linux (you prob don't) just use terraos or shimboot

**Comment by SniperWasTaken** [comment]
Sadly, I would recommend badrecovery, but that unfortunately says it needs your kernver to be 3 and under.

However, IF you have **COURAGE** and willing to **TEAR APART AND TAKE THE RISK OF DAMAGING YOUR CHROMEBOOK**, I'd recommend [The Tsunami](https://blog.darkn.bio/blog/3-the-tsunami)
